rllib/evaluation/sampler.py

```diff
@@ -1172,11 +1175,26 @@ def _do_policy_eval(
             policy: Policy = _get_or_raise(policies, policy_id)
 
         input_dict = sample_collector.get_inference_input_dict(policy_id)
-        eval_results[policy_id] = policy.compute_actions_from_input_dict(
-            input_dict,
-            timestep=policy.global_timestep,
-            episodes=[active_episodes[t.env_id] for t in eval_data],
-        )
+        # New Policy API: Only method to ever use (and override) to get and compute
```

Comment:
Sampler has to distinguish between `compute_actions` (PolicyV2s) and `compute_actions_from_input_dict` (old Policy classes).